docs/layouts/alias.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html{{ with site.LanguageCode | default site.Language.Lang }} lang="{{ . }}"{{ end }}>
+  <head>
+    <title>{{ .Permalink }}</title>
+    <link rel="canonical" href="{{ .Permalink }}">
+    <meta name="robots" content="noindex">
+    <meta charset="utf-8">
```

Comment:
```suggestion
    <meta charset="utf-8">
    <title>{{ .Permalink }}</title>
    <link rel="canonical" href="{{ .Permalink }}">
    <meta name="robots" content="noindex">
```

Probably mostly superficial, but I prefer `charset` first so the browser can immediately switch into "utf-8" mode.

Reply:
The browser barely even loads this page before redirecting, does the charset do much?

Reply:
Likely not, it's just a habit that I follow so stuff like `<title>` does not parse weirdly when the browser was not yet told the charset to use. Likely makes zero difference.

Reply:
As per [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta#charset):

> <meta> elements which declare a character encoding must be located entirely within the first 1024 bytes of the document.

So there is a theoretical possibilty it moves outside this boundary when the title is 1kB long 😆